server/lib/pbench/test/unit/test_process_tb.py

```diff
@@ -0,0 +1,289 @@
+import os
+import logging
+import pytest
+
+from pathlib import Path
+
+from pbench.process_tb import ProcessTb, Results
+from pbench.test import tarball, bad_tarball, fixture_path
+
+
+class MockConfig:
+    TS = "run-1970-01-02T00:00:00.000000"
+
+    def get(section: str, option: str):
+        pass
```

Comment:
I think that LGTM would like to see this method return a value.  Try replacing the `pass` with `return ""` and see if that satisfies it.

Reply:
Or...not?  Is `get()` a class method or an instance method??

Reply:
The intention is to use a `MockConfig` instance as a replacement for a `PbenchConfig` instance; `PbenchConfig.get()` is an instance function, and [its signature](https://github.com/riya-17/pbench/blob/ebb170c5425ec38fd3eded13e6e1d8120f9637b3/server/lib/pbench/__init__.py#L309) looks like this:
```
    def get(self, *args, **kwargs):
```
So, the stub function should have a `self` parameter, as should all the mocks.  Ideally, the stub and all the mocks would also have signatures which match `PbenchConfig.get()` as well, but I suspect that that is going to be annoying in a couple of ways.  🙁  Perhaps, just adding the `self` parameter will suffice.

Reply:
That did satisfy it. Thanks webb!

Reply:
I will add it

Reply:
I am unable to understand why adding `self` as first parameter of mock_get function signature is making it fail.

Reply:
I think I figured it out!  In some of the calls to the `ProcessTb` constructor, you're passing the `MockConfig` **_class_** object instead of invoking _its_ constructor to produce an instance object.  I presume that invoking the `get()` method on the class object binds to a totally different function from the one you would get if you invoked it on an instance of the class.

Reply:
Once we get everything else smoothed out, I think I have a better approach to mocking the `PbenchConfig` object, which wouldn't have tripped you up like this.  But, let's put that off for the moment while we get everything else settled.